doc/decisions/0_drafts/internal_cache.md

```diff
@@ -1,185 +0,0 @@
-# Internal KeySet Cache
```

Comment:
Did you use `git mv` to move this file?

Reply:
No, Nautilus.

Reply:
`git mv` doesn't actually do anything special (see https://stackoverflow.com/a/1094392). The file is no longer detected as moved, because too much has changed.

Reply:
So we should separate the moving of the file and changing the content? Or is it a GitHub problem and moves are detected differently than `git` would do?

a1e7bf8aafa711d719aad8693cad99eeb5b3c1f3

Reply:
Also locally the moved file still has all the history of the "old" file.

Reply:
Seems like a GitHub UI issue in the PR UI. I think the treat the whole PR as a single commit to generate the diff. If you look at the git blame of the moved file, the old history is still there: https://github.com/ElektraInitiative/libelektra/blame/330dc5383aedca4aa0c935e9abdd93f3868203ac/doc/decisions/1_in_discussion/internal_cache.md